src/components/TeamMemberTasks/TeamMemberTasks.jsx

```diff
@@ -447,6 +447,7 @@ const TeamMemberTasks = React.memo(props => {
                   return (
                     <TeamMemberTask
                       user={user}
+                      userPermission={props?.auth?.user?.permissions?.frontPermissions?.includes('putReviewStatus')}
```

Comment:
This is unrelated to this PR. 
Also, it should be removed in general to remove prop drilling. Call `hasPermission()` in the component where you actually need this permission instead. `hasPermission()` will also check for role permissions in addition to user permissions.